apps/prairielearn/src/pages/partials/questionScorePanel.ejs

```diff
@@ -13,12 +13,15 @@
       </tr>
       <% } %>
       <% if (assessment.type == 'Homework') { %>
-      <tr>
-        <td colspan="2" class="text-wrap">
-          Previous variants:
-          <%- include('../partials/questionAwardedPoints', {instance_question_id: instance_question.id, previous_variants: instance_question_info.previous_variants});  %>
-        </td>
-      </tr>
+        <%# Only show previous variants if the question allows multiple variants, and there is at least one variant other than the current one %>
+        <% if (instance_question_info.previous_variants?.some((previous_variant) => previous_variant.id != locals.variant?.id)) { %>
```

Comment:
I think the comment is now slightly wrong (we don't require the question to allow multiple variants).